dev-util/mingw64-runtime/mingw64-runtime-10.0.0-r1.ebuild

```
@@ -76,6 +76,7 @@ src_configure() {
 	local conf=(
 		--prefix="${prefix}"
 		--libdir="${prefix}"/lib
+		$(usex default-ucrt "--with-default-msvcrt=ucrt" "")
```

I'll change it myself on merge, but note you can do this shorter form in EAPI=8 for same results:
```suggestion
		$(usev default-ucrt --with-default-msvcrt=ucrt)
```